### ISSUE TITLE
Add access keys (mnemonics)

### ISSUE THREAD

**Issue by Otiel** [ForkIssues/TrackerWin#320]
Can you please add access keys (character used in combination with <kbd>Alt</kbd> to activate a control) to all menus and relevant controls?

This will improve productivity for those mainly using the keyboard.

**Comment by SteveALee** [comment]
I also find this lack extremely annoying. 

The menu Access Key / mnemonics are a very old win 32 standard (though Office breaks with the tradition with the Ribbon etc). I learnt Windows programming with 3.0 for workgroups so have very old habits :)

VS Code does it right (except `Alt` by itself does not show the menubar)
eg Alt + F drops down the File menu with mnemonics underlined on the bar and dropdown.

![image](https://user-images.githubusercontent.com/618922/66640826-8e72d400-ec11-11e9-844b-3932291d56e4.png)

For more info search for `Access Keys` in 
[Menus](https://docs.microsoft.com/en-gb/windows/win32/uxguide/cmd-menus)
[Keyboard](https://docs.microsoft.com/en-gb/windows/win32/uxguide/inter-keyboard)

**Comment by SteveALee** [comment]
It's not JUST the menus. Access keys / mnemonics should be available for all controls in the UI, esp dialogs. Am surprised if the .NET UI lib you are using doesn't provided for these as such a basic feature. Thanks

**Comment by DanPristupov** [comment]
> VS Code does it right (except Alt by itself does not show the menubar)

I like that the access keys are hidden by default. I'll try to find out how this is implemented.

**Comment by SteveALee** [comment]
In my win32 days you just plonked a `&` before the mnemonic char in the menu item or dialog lable definition and it just all worked :)

**Comment by Otiel** [comment]
>In my win32 days you just plonked a & before the mnemonic char in the menu item

That's for winforms. In WPF, the character to use is the underscore `_`.

**Comment by SteveALee** [comment]
> That's for winforms.

well before that - C API and resource editors :)

**Comment by Otiel** [comment]
> I like that the access keys are hidden by default. I'll try to find out how this is implemented.

@DanPristupov What do you mean? 

The Windows setting ("Underline access keys when available") sets this behavior:
* If "Off" → access keys are not underlined by default, but are underlined when you hold "Alt".
* If "On" → access keys are always underlined.

I don't see how VS Code behaves differently than any other program.

**Comment by DanPristupov** [comment]
@Otiel 

Ouch, sorry! I made a mistake quoting VSCode instead of Visual Studio.

Here's what I mean:

Usual menu click in Visual Studio doesn't show the underscored:
<img width="393" alt="Screenshot 2019-10-15 at 15 39 53" src="https://user-images.githubusercontent.com/618115/66836659-0ae31b00-ef62-11e9-9797-614edc6d26fa.png">

However Alt+V (and any other Alt+key combination) makes them appear:
<img width="393" alt="Screenshot 2019-10-15 at 15 40 45" src="https://user-images.githubusercontent.com/618115/66836735-2a7a4380-ef62-11e9-8a1e-5358be42fe77.png">

**Comment by Otiel** [comment]
@DanPristupov Again, that seems standard. Or I don't see what's special here?? 😕

**Comment by SteveALee** [comment]
@DanPristupov 

This is absolutely standard behaviour and has been since at least win 3.0. Also if you press and release ALT the mnemonic underlines appear, press again they dissapear (try notepad, the simplest and probably most standard app from win32 days as it's basically an edit control in a window with some menus.).

In addition if the menu bar is hidden the ALT key should it make appear with the menmonics displayed.

I just noticed Firefox has it a bit wrong as the menemonics are always displayed for me.

Also Windows Explorer doesn't now use underscores but shows the mnemonic in a square block. At least in my WIndows Insiders build. Interesting.

**Comment by NN---** [comment]
@Otiel
> I like that the access keys are hidden by default. I'll try to find out how this is implemented.

If you are in Windows, then this is the setting:
![image](https://user-images.githubusercontent.com/580536/68369687-97d55a80-0143-11ea-952f-cb1205c0acd0.png)

**Comment by Otiel** [comment]
@NN--- Yep, that's what I described in my comment [here](https://github.com/ForkIssues/TrackerWin/issues/320#issuecomment-542213602).

**Comment by Otiel** [comment]
Any chance to see this implemented soon?

**Comment by Otiel** [comment]
More than one year since I've nagged you on this, this calls for a reminder. 😁

This would really benefit those that use the keyboard more than the mouse, is there any chance we can get this?

**Comment by OZ1** [comment]
It is available in the main menu, but context menus still lack such mnemonics.
